scarlet/transformations.py

```diff
@@ -538,18 +426,18 @@ def getSymmetryOp(shape):
     acts on the flattened image to return its symmetric version.
     """
     size = shape[0]*shape[1]
-    name = "Symm"
+    name = "getSymmetryOp"
     key = tuple(shape)
     try:
-        symmetryOp = check_cache(name, key)
+        symmetryOp = Cache.check(name, key)
     except KeyError:
         idx = np.arange(shape[0]*shape[1])
         sidx = idx[::-1]
-        symmetryOp = getIdentityOp(shape)
+        symmetryOp = getIdentityOp(shape).L
         symmetryOp -= scipy.sparse.coo_matrix((np.ones(size),(idx, sidx)), shape=(size,size))
-        symmetryOp = LinearOperator(symmetryOp)
-        global cache
-        cache[name][key] = symmetryOp
+        symmetryOp = proxmin.utils.MatrixAdapter(symmetryOp, axis=1)
+        _ = symmetryOp.spectral_norm
```

Comment:
I think you don't need the `_ = ` here, and this can just be
```
symmetryOp.spectral_norm
```
to calculate the spectral norm.